### PR TITLE
plugin.video.rtk 1.0.0

### PR DESCRIPTION
**Pull request by hkastrati7**
### Description
This is the official add on of Radio Television of Kosovo  for Kodi, in which you can watch four TV channels and can listen two of radio stations. Enjoy!
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a sinlge commit with using the following style: [plugin.video.foo] v1.0.0

Addtional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuesly monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.rtk/default.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+import sys
+import xbmcgui
+import xbmcplugin
+
+addon_handle = int(sys.argv[1])
+
+xbmcplugin.setContent(addon_handle, 'videos')
+
+li = xbmcgui.ListItem('RTK1', iconImage=iconImageRtk1)
```
Comment (learningit) [inline plugin.video.rtk/default.py]: In a future update, please remove the init parameter iconImage and use listItem.setArt(). The init is currently deprecated. Please see http://mirrors.kodi.tv/docs/python-docs/16.x-jarvis/xbmcgui.html#ListItem-setArt for info.

File: plugin.video.rtk/addon.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rtk"
+       name="RTK"
+       version="1.0.0"
+       provider-name="RTK">
+  <requires>
+    <import addon="xbmc.python" version="1.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
```
Comment (learningit) [inline plugin.video.rtk/addon.xml]: Please set xbmc.python to "2.24.0" for Jarvis. Pls see http://kodi.wiki/view/addon.xml#Dependency_versions for details.

**Comment by learningit** [comment]
So, I can merge after you add a commit for fixing the xbmc.python and <source> issues listed above. Rest can wait for next major update you make. Pls name the new commit exactly the same as the current one. Thanks

File: plugin.video.rtk/addon.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <language>sq</language>
+    <summary lang="en">Watch RTK channels and radio stations</summary>
+    <summary lang="sq">Shikoni kanalet e RTK-së si dhe radio stacionet</summary>
+    <description lang="en"> RTK offers two radio stations broadcasting a diverse programming of news and music and four 24-hour television services broadcasting on terrestrial and satellite networks.</description>
+    <description lang="nb"> RTK ofron transmetimin e llojllojshmërisë së programeve muzikore dhe lajmeve përmes dy radiostacioneve sidhe shërbimin e transmetimit të kanaleve televizive 24 orë në rrjetin tokësor dhe satelitor.</description>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+    <forum></forum>
+    <source></source>
```
Comment (learningit) [inline plugin.video.rtk/addon.xml]: Please add source url to github

**Comment by hkastrati7** [comment]
I made the commit, first I forgot to name it, then I named it plugin.video.rtk 1.0.0, and I realized that the first one was with brackets,like this [plugin.video.rtk] 1.0.0
If it is a problem, tell me and I will change

**Comment by learningit** [comment]
I squashed the commits and merged into the repo. In the future, please squash commits into a single commit when submitting. Instructions to do that are here : https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

We welcome you to create a support page in our forum - forum.kodi.tv  and a wiki page which you can update at kodi.wiki will be auto-created in a day or two. Thanks

**Comment by hkastrati7** [comment]
Thanks for the commits. Now it's something that its not clear for me yet.
I wanted to set my add On in this list, so when I open Kodi in desktop, and go to Video/Add-ons and click "Get more" , RTK to be in that list. For now I can't see it there

**Comment by learningit** [comment]
At a scheduled time  (usually once every day or so) a batch is run against the repo to copy all updated addons to (in your case) http://mirrors.kodi.tv/addons/jarvis/  it then can take up to a day for all the global mirrors to update/synch.  It will then be available within the Kodi repository you mention above.  Another script will run against the mirror (maybe repo? don't remember right now) which which will update addons.kodi.tv. Your addon will be somewhere in here: http://addons.kodi.tv/category/video/ when that happens.

**Comment by hkastrati7** [comment]
I thinks this is what I was looking for.
Thanks a lot

**Comment by learningit** [comment]
You can see that your add-on is in the repo at https://github.com/xbmc/repo-plugins/tree/jarvis/plugin.video.rtk

Which when you look at it, you will realize that you will want to put some more info into readme.md next time

**Comment by hkastrati7** [comment]
Yes, thanks for advice. Next time I will write an introduction for the add on i that file

**Comment by MartijnKaijser** [comment]
@hkastrati7 seems your addon.xml fails XML syntax check. Could you please fix this and send an update (including addon version bump)?

**Comment by MartijnKaijser** [comment]
@hkastrati7 i've already fixed it. Please make sure you fix it on your end as well for next update
https://github.com/xbmc/repo-plugins/commit/01bff5502ed18e5da415f399adb3a592b77f758b

**Pull request merged**